packages/homepage/src/config/hiring.js

```diff
@@ -0,0 +1,2 @@
+// Should probably be fetched from recruitee, but it's hard coded for now to avoid an extra request.
+export const OPEN_JOBS_COUNT = 3;
```

Comment:
You could use this implementation here to fetch the amount of open positions: https://github.com/codesandbox/codesandbox-client/blob/master/packages/homepage/src/pages/jobs/index.js#L33

Reply:
Yeah this was my initial thought too, then I realised it would be firing off three requests on every page. I think we need to start using something like SWR or ReactQuery to make requests like this with caching to avoid too many network requests. WDYT?

Reply:
Agreed, it makes sense. A React-context would also work to avoid extra requests, then it wouldn't increase the bundle with one more package in the application.

Reply:
Makes sense to me! Another thing we could do is a cache like this, if it's easier (I'm not familiar enough with react-context):

```ts
let jobsFetchRequest: Promise<Job[]> | null = null;

function fetchJobs() {
  jobsFetchRequest = jobsFetchRequest || fetch(...);

  return jobsFetchRequest;
}

```

But that would be more hacky, though I'm not sure if we'll get more cases where we need caching 🤔 .

Reply:
[Task](https://linear.app/codesandbox/issue/CSB-980/fetch-job-count-from-recruitee) for later